### PR TITLE
Add support for mirror mode

### PR DESCRIPTION
**Pull request by lubosmj**
closes #5771
https://pulp.plan.io/issues/5771

### CODE REVIEW

File: CHANGES/5771.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Allow users to sync content in mirror mode
```
Comment (ipanova) [inline CHANGES/5771.feature]: s/allow/enable

File: docs/workflows/sync.rst [hunk 2]
```diff
@@ -75,6 +75,11 @@ sync with. You are telling pulp to fetch content from the remote and add to the
 .. literalinclude:: ../_scripts/sync.sh
    :language: bash
 
+.. note::
+    In the above example, the payload contains the field ``mirror=False``. This means that the
+    sync will be run in the additive mode only. Set ``mirror`` to ``True`` if Pulp should pull
```
Comment (ipanova) [inline docs/workflows/sync.rst]: s/if Pulp should/and Pulp will/

File: docs/workflows/sync.rst [hunk 3]
```diff
@@ -75,6 +75,11 @@ sync with. You are telling pulp to fetch content from the remote and add to the
...
+    in new content and delete content which was also deleted from upstream.
```
Comment (ipanova) [inline docs/workflows/sync.rst]: s/delete/remove

File: CHANGES/5771.feature (hunk 1 above)
Comment (ipanova) [reply CHANGES/5771.feature]: @lubosmj please add this to the Feature list also

Comment (lubosmj) [reply CHANGES/5771.feature]: I believe It is already mentioned in the feature list, see the first bullet point https://github.com/pulp/pulp_container/blob/master/docs/index.rst#features

Comment (ipanova) [reply CHANGES/5771.feature]: it is a bit confusing, mirror in that context means - sync, you can mirror in additive mode and mirror mode. We can re-phrase - Sync container image repositories hosted on Docker-hub,...in mirror or additive mode,

**Pull request merged**